workspaces/adventure-pack/goodies/python3/src/list_transpose/__init__.py

```diff
@@ -0,0 +1,16 @@
+from typing import Any
+
+
+def list_transpose(list: list[Any]) -> list[Any]:
```

Comment:
Same comment as in #370 about `Any`. In this case I actually think we could be more concrete. For starters, the input should be a list of lists, so we could express that in the type annotation.

This might also be an appropriate place to use generics. I'm new to Python typing but https://mypy.readthedocs.io/en/stable/generics.html#generic-functions seems like a promising read.